src/http/client.ts

```diff
@@ -2,6 +2,9 @@ import axios from 'axios';
 import { HttpClientConfig, HttpInstance } from '../types';
```

Comment:
Having had some time to think about it, this does change a few extra things I belief 🤔 The non authenticated `http` client is used in some other calls that would now also have their timestamp changed. **Might be nothing**, but just want to check if we're aware of this. User Service seems to be fine, just looking at the oAuth1/2/proxy internal calls now, which do seem to be affected

Reply:
I don't see any changes besides the one interface for the OAuth1TokenData boi? Did you notice anything I missed? 🤔

Reply:
I indeed think that is all 🤔